src/video/wayland/SDL_waylandvideo.c

```diff
@@ -632,6 +636,14 @@ display_handle_global(void *data, struct wl_registry *registry, uint32_t id,
                 &qt_windowmanager_interface, 1);
         qt_windowmanager_add_listener(d->windowmanager, &windowmanager_listener, d);
 #endif /* SDL_VIDEO_DRIVER_WAYLAND_QT_TOUCH */
+
+#ifdef SDL_VIDEO_DRIVER_WAYLAND_TABLET
+    } else if (SDL_strcmp(interface, "zwp_tablet_manager_v2") == 0) {
```

Comment:
This can be moved about the Qt block above

Reply:
> about

about? you mean over?